File: CS/WindowsFormsApplication21/CustomDataNavigatorButtons.cs
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication21 {
    public class CustomDataNavigatorButtons : DataNavigatorButtons {
        public CustomDataNavigatorButtons(INavigatorOwner owner)
            : base(owner) {

        }
        protected override NavigatorButtonCollectionBase CreateNavigatorButtonCollection() {
            return new CustomDataNavigatorButtonCollection(this);
        }
    }
}

File: VB/WindowsFormsApplication21/CustomDataNavigatorButtons.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication21
	Public Class CustomDataNavigatorButtons
		Inherits DataNavigatorButtons
		Public Sub New(ByVal owner As INavigatorOwner)
			MyBase.New(owner)

		End Sub
		Protected Overrides Function CreateNavigatorButtonCollection() As NavigatorButtonCollectionBase
			Return New CustomDataNavigatorButtonCollection(Me)
		End Function
	End Class
End Namespace
